### PR TITLE
Sync manifests with upstream for 0.11.0-alpha

### PR DESCRIPTION
**Pull request by VedantMahabaleshwarkar**
manifest sync for https://github.com/opendatahub-io/odh-manifests/commit/4cd5697964912896c3d482d372f9ec17a3c5ae42

### CODE REVIEW

File: model-mesh/dependencies/quickstart.yaml [hunk 1]
```diff
@@ -49,7 +49,7 @@ spec:
             defaultMode: 0554
       initContainers: 
         - name: etcd-secret-creator
-          image: registry.redhat.io/openshift4/ose-cli@sha256:25fef269ac6e7491cb8340119a9b473acbeb53bc6970ad029fdaae59c3d0ca61
+          image: quay.io/openshift/origin-cli
```
Comment (Jooho) [inline model-mesh/dependencies/quickstart.yaml:52]: this one should be registry.redhat.io

Comment (VedantMahabaleshwarkar) [reply model-mesh/dependencies/quickstart.yaml]: fixed

**Review by taneem-ibrahim — APPROVED** [review]
lgtm

**Pull request merged**